driver-kafka/src/main/java/io/openmessaging/benchmark/driver/kafka/KafkaBenchmarkDriver.java

```diff
@@ -92,44 +91,94 @@ public String getTopicNamePrefix() {
     public CompletableFuture<Void> createTopic(String topic, int partitions) {
         return CompletableFuture.runAsync(() -> {
             try {
-                admin.createTopics(Arrays.asList(new NewTopic(topic, partitions, config.replicationFactor))).all()
-                        .get();
+                ListTopicsResult allTopics = admin.listTopics();
```

Comment:
I think I can actually revert this back to what you had because you're doing the wait outside this function...it currently creates the topics with no issues with this here.